libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -80,7 +80,7 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
     // @Param: INJECT_TO
     // @DisplayName: Destination for GPS_INJECT_DATA MAVLink packets
     // @Description: The GGS can send raw serial packets to inject data to multiple GPSes.
-    // @Values: 0:send to first GPS, 1:send to 2nd GPS, 127:send to all
+    // @Values: 0:send to first GPS,1:send to 2nd GPS,127:send to all
```

Comment:
Having the spaces or not having them? We are terribly inconsistent, and it
made me add strip()'s to everywhere before parsing.

On Thu, Jan 7, 2016 at 12:00 AM, Tom Pittenger <notifications@github.com>
wrote:

> In libraries/AP_GPS/AP_GPS.cpp
> <https://github.com/diydrones/ardupilot/pull/3450#discussion_r49046547>:
>
> > @@ -80,7 +80,7 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
> >      // @Param: INJECT_TO
> >      // @DisplayName: Destination for GPS_INJECT_DATA MAVLink packets
> >      // @Description: The GGS can send raw serial packets to inject data to multiple GPSes.
> > -    // @Values: 0:send to first GPS, 1:send to 2nd GPS, 127:send to all
> > +    // @Values: 0:send to first GPS,1:send to 2nd GPS,127:send to all
>
> I think this "@Values <https://github.com/Values>" format breaks the
> stuff @DonLakeFlyer <https://github.com/DonLakeFlyer> is doing
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diydrones/ardupilot/pull/3450/files#r49046547>.
>